client/ReviewList.jsx

```diff
@@ -123,11 +123,47 @@ class ReviewList extends React.Component {
     }
     const urlArray = url.split('/');
     const endpoint = urlArray[urlArray.length - 2];
+    /*
+      URL array === http://localhost:2625/item
+    */
     this.setState({
       url: urlArray.slice(0, urlArray.length - 2).join('/')
     });
+
```

Comment:
Please let me know if you think this will work: setting state with a review array received from one GET request. I believe if the review objects have the necessary properties it should be fine.